nx-X11/programs/Xserver/os/utils.c

```diff
@@ -141,7 +141,7 @@ OR PERFORMANCE OF THIS SOFTWARE.
 #endif
 
 #ifdef XKB
-#include <nx-X11/extensions/XKBsrv.h>
+#include <xkbsrv.h>
```

Comment:
same here... (I won't mark any more of these, please make sure to fix all occurrences)